conbench/entities/history.py

```diff
@@ -402,14 +406,11 @@ def _query_and_calculate_distribution_stats(
     # Find all historic results in the distribution to analyze
     history = (
         Session.query(
-            BenchmarkResult.case_id,
-            BenchmarkResult.context_id,
-            BenchmarkResult.change_annotations,
-            BenchmarkResult.mean,
-            BenchmarkResult.timestamp.label("result_timestamp"),
-            Hardware.hash,
+            # we can use the `defer` method to not select all columns
+            BenchmarkResult,
+            Hardware.hash.label("hardware_hash"),
             s.sql.expression.literal(baseline_commit.repository).label("repository"),
-            commits.c.ancestor_timestamp.label("timestamp"),
+            commits.c.ancestor_timestamp.label("commit_timestamp"),
```

Comment:
Love this; we should carry this disambiguation throughout, e.g. on L458 (which arguably should be `result_timestamp`, though I'm not sure if it currently is)